### PR TITLE
SmartDashboard buttons

### PR DESCRIPTION
**Pull request by MajorDarius**
Included the new commands to OI and added buttons for the SmartDashboard for the new commands

### CODE REVIEW

File: src/OI.cpp [hunk 1]
```diff
@@ -43,7 +47,11 @@ OI::OI()
 
   SmartDashboard::PutData("TankDrive", new TankDrive());
   SmartDashboard::PutData("Autonomous Command", new AutonomousCommand());
-
+  SmartDashboard::PutData("FeederOn", new ConveyorControl(1.0, 0.0));
+  SmartDashboard::PutData("MoverOn", new ConveyorControl(0.0, 1.0));
+  SmartDashboard::PutData("IntakeOn", new SetIntake(1.0));
+  SmartDashboard::PutData("ShooterOn", new SetShooter(1.0));
+  SmartDashboard::PutData("TurretOn", new SetTurret(1.0));
```
Comment (duane701) [inline src/OI.cpp]: How do we turn them off?

File: src/OI.cpp [hunk 2]
```diff
@@ -43,7 +47,11 @@ OI::OI()
 
   SmartDashboard::PutData("TankDrive", new TankDrive());
   SmartDashboard::PutData("Autonomous Command", new AutonomousCommand());
-
+  SmartDashboard::PutData("FeederOn", new ConveyorControl(1.0, 0.0));
```
Comment (duane701) [inline src/OI.cpp]: The string "FeederOn" in this case is used in the SmartDashboard user interface as a title for the control. I think you can actually have spaces and use real English and capitalization and everything. 😈 

Perhaps "Feeder On", "Mover On"? or is that just weird?

**Review by duane701 — CHANGES_REQUESTED** [review]
FTP

File: src/OI.cpp (hunk 1 above)
Comment (Diego-Andrade) [reply src/OI.cpp]: You cancel the command on the Smart Dashboard. If you haven't seen the buttons, they initially have a Start button next to them, and once click it transforms to a Cancel.

Comment (duane701) [reply src/OI.cpp]: Thanks @Diego-Andrade. 

Walking through the process ... 

1. Press Start. The command starts running on this Talon. The button changes to Cancel.
2. Press Cancel.
  a. The current command is interrupted and exits.
  b. The default command is instantiated and restarted.

But wait ... most of these subsystems have no default command. Due to the implementation of the Talon, the previous setting keeps running. 

I think we should have taken your (@Diego-Andrade) early suggestion to write default commands for all of these systems. 

Do we have default commands for each of these subsystems that disables the talons (ie set the motor to 0?).

**Comment by duane701** [comment]
Thank you for the fixup commit. However, fixup is specifically spelled `fixup!`. No mixed case. So close. 😜

**Comment by duane701** [comment]
LGTM.

I checked the subsystems referenced here all have default commands. `Lifter` is the only subsystem missing a default command. We can fix that in future work when we know more about the lifter.

I will handle the rebase and push a new branch. If someone else can do the PR, I can approve it.

**Comment by duane701** [comment]
See new branch `feature/SmartDashBoard.v1`

**Pull request closed without merging**